src/store/mutations.js

```diff
@@ -182,10 +182,24 @@ function resetRequests(state) {
 }
 
 function setAcceptCommitInfo(state, value) {
+  if (value === false && storage.get('commitInfoAccepted')) {
```

Comment:
Same as below, don’t need to use `storage.get` here use the in memory value instead.

Reply:
See response/question above ^